internal/eventstore/v3/field.go

```diff
@@ -143,7 +143,16 @@ func buildSearchCondition(builder *strings.Builder, index int, conditions map[ev
 	return args
 }
 
-func handleFieldCommands(ctx context.Context, tx database.Tx, commands []eventstore.Command) error {
+func (es *Eventstore) handleFieldCommands(ctx context.Context, tx database.Tx, commands []eventstore.Command) error {
+	// CockroachDB by default does not allow multiple modifications of the same table using ON CONFLICT
+	// Thats why we enable it manually
+	if es.client.Type() == "cockroach" {
+		_, err := tx.ExecContext(ctx, "SET enable_multiple_modifications_of_table = on")
```

Comment:
Does this SET needs to be send in every TX? When I read https://www.cockroachlabs.com/docs/v24.3/set-vars it should be valid for the duration of the session (eg Connection). IMO we can throw this in another `AfterConnect` hook so that it's a roundtrip after connect once and safe the repeated roundtrip for every transaction